go/client/cmd_ctl_stop.go

```diff
@@ -9,12 +9,19 @@ import (
 	"github.com/keybase/cli"
 	"github.com/keybase/client/go/libcmdline"
 	"github.com/keybase/client/go/libkb"
+	"os"
 )
 
 func NewCmdCtlStop(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
 	return cli.Command{
 		Name:  "stop",
 		Usage: "Stop the background keybase service",
+		Flags: []cli.Flag{
+			cli.BoolFlag{
```

Comment:
we've only implemented this for windows, so for the other platforms, we shouldn't allow this flag.